### DISCUSSION TITLE
使用trz上传大文件，传不上去

### DISCUSSION THREAD

**Discussion by czxin788** [electerm/electerm#3240] [Q&A]
使用trz上传大文件，传不上去
版本：1.37.110
![image](https://github.com/electerm/electerm/assets/13861904/ac9b64bd-4a75-4483-8135-ef2a86cd9643)

**Comment by zxdong262** [comment]
请问多大的文件

**Reply by czxin788** [reply]
1.6G
···
-rw-rw-rw- 1 root root 1.6G Jan  6 13:46 datax.tar.gz
···

**Comment by zxdong262** [comment]
最新版可以试试吗
